src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -1256,22 +1256,25 @@ def test_WINNF_FT_S_FDB_6(self, config_filename):
         grant_ids_g1.append(resp['grantId'])
         cbsd_ids_g1.append(resp['cbsdId'])
 
-    # Step 6: Send relinquishment request for grant 'G1'
-    relq_requests = {'relinquishmentRequest': []}
-    for cbsd_id, grant_id in zip(cbsd_ids_g1, grant_ids_g1):
-      # Update Relinquishment Request content
-      relq_requests['relinquishmentRequest'].append({
-        'cbsdId': cbsd_id,
-        'grantId': grant_id
-      })
-    relq_responses = self._sas.Relinquishment(relq_requests)['relinquishmentResponse']
-    self.assertEqual(len(relq_responses), len(relq_requests['relinquishmentRequest']))
+    # If SAS UUT Responded with SUCCESS for Grant then Proceed with Relinquishment or else skip it
+    if len(grant_ids_g1):
```

Comment:
I think you can simplify with just `if grant_ids_g1` but of course this is also correct.